### PR TITLE
Changelog and preparation for collectd 5.10

### PR DESCRIPTION
**Pull request by mrunge**
This patch provides the changelog and the change to version-gen.

### CODE REVIEW

**Comment by mrunge** [comment]
This is the last missing piece before I'd tag 5.10.0

**Comment by mrunge** [comment]
Yes, sure!

File: contrib/redhat/collectd.spec [hunk 1]
```diff
@@ -2929,6 +2934,9 @@ fi
 %doc contrib/
 
 %changelog
+* Fri Oct 18 2019 Matthias Runge <mrunge@redhad.com> - 5.10.0-1
```
Comment (rubenk) [inline contrib/redhat/collectd.spec]: redhat I presume?

Comment (mrunge) [reply contrib/redhat/collectd.spec]: damn, yes!

**Comment by mrunge** [comment]
Last time I checked, CI was passing but the result was not propagated back here, let's see with this commit

**Comment by rpv-tomsk** [comment]
Still playing this game?
Still waiting for reviewers? You is untrusted.

**Comment by faxm0dem** [comment]
Let's solve the source tarball location first please. I don't want another botched release like the infamous 5.9.x series:

* Do we change the source location to github in specfile?
* Who/what created the source tarball on collectd.org/files?
* It's the release process 200% ok? Will the tarball be tagged to correct version and contain all generated files?

**Comment by mrunge** [comment]
I've changed the source location to github, since I don't have access to collectd.org location. I can only assume, there is a copy job to fetch the file from github.com.

Last time, there was an issue with the tarball, since I didn't follow the release process as documented in the wiki https://collectd.org/wiki/index.php/Release_process
I've fixed the tarball on github, but could not do the same for collectd.org.
Makes sense?

**Comment by rpv-tomsk** [comment]
I do not consider it possible to continue work on the project when there is no contact with the founder and there are a lot of technical limitations and defects (errors on the project’s website for a long time, there is no way to fix them, permanent errors in CI and no way to fix them, impossibility to modify the project codebase by a "trusted" maintainer, repeated inconsistent code style rules changes).

Hope, Matthias will wait for several months before "reviewer with write access" will take a look on this PR. I wish him to get as much as possible enjoy from this and similar waitings.

I don't understand, how it is possible to maintain project in these conditions. 

Matthias' efforts deserve respect, but I do not share his position to try to go against these obstacles.

**Comment by mrunge** [comment]
@rubenk @faxm0dem 
Would you be fine with merging this? Any steps missing?

**Comment by rpv-tomsk** [comment]
How is your feeling to be untrusted maintainer?

**Comment by rpv-tomsk** [comment]
Matthias, your position is a bit strange.

You waiting for approve from somebody, while you do not approve their PR's too.

Want to notice, that is Collectd community main feature. 
Nobody wants to work on other's changes, only on their own. 

When maintainers (more precisely, the developers) has permissions to merge own PRs, project has [slowly] developing. 
When maintainer (developer) has not permissions to merge own PRs -  - development has stopped due to additional obstacles.

**Comment by rubenk** [comment]
@mrunge I think it's alright and already gave my approval, but since you've added @octo for review this needs his approval to to merge.

**Comment by mrunge** [comment]
@rubenk thank you, I removed @octo from the review. You may want to approve (again)?

**Comment by faxm0dem** [comment]
I'll test this branch tomorrow and approve your changes when done. Thanks for addressing my issues

**Pull request merged**